src/ui/hexeditwidget.cc

```diff
@@ -302,6 +317,21 @@ void HexEditWidget::initParsersMenu() {
   }
 }
 
+void HexEditWidget::initUnprintablesMenu() {
+  unprintables_menu_.clear();
+
+  HexEdit::UnprintablesMode modes[] = {HexEdit::UnprintablesMode::Dots,
+                                       HexEdit::UnprintablesMode::Windows_1250};
+
+  for (auto mode : modes) {
+    QAction* action = new QAction(hex_edit_->unprintablesModeToString(mode),
+                                  &unprintables_menu_);
+    connect(action, &QAction::triggered,
+            [=]() { this->hex_edit_->setUnprintablesMode(mode); });
```

Comment:
Please use `[this, mode]`, it's less error-prone and more explicit.

Reply:
Done.